src/main/java/com/wix/mysql/config/SchemaConfig.java

```diff
@@ -14,14 +15,14 @@
     private final String name;
     private final Charset charset;
     private final List<File> scripts;
+    private final List<String> commands;
 
-    private SchemaConfig(
-            final String name,
-            final Charset charset,
-            final List<File> scripts) {
+    private SchemaConfig(String name, Charset charset, List<File> scripts, List<String> commands)
+    {
```

Comment:
please fix formatting - given you use intellij with defaults - you can just autoformat code, should be fine.